--- config.def.h	2019-01-11 12:46:04.789213233 -0500
+++ config.h	2019-01-14 10:58:00.303533360 -0500
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "Iosevka Term:size=10:antialias=true:autohint=true";
+static int borderpx = 5;

 /*
  * What program is execed by st depends of these precedence rules:
@@ -83,40 +83,40 @@
 unsigned int tabspaces = 8;

 /* Terminal colors (16 first used in escape sequence) */
-static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-};
-
+static const char *colorname[] =
+  {
+   /* 8 normal colors */
+   "#282828",
+   "#9d1306",
+   "#79740e",
+   "#b57614",
+   "#076678",
+   "#8f3f71",
+   "#00a7af",
+   "#bdae93",
+
+   /* 8 bright colors */
+   "#686868",
+   "#fb4933",
+   "#b8bb26",
+   "#fabd2f",
+   "#83a598",
+   "#d3869b",
+   "#3fd7e5",
+   "#fdf4c1",
+
+   [255] = 0,
+
+   /* more colors can be added after 255 to use with DefaultXX */
+   "#cccccc",
+   "#555555",
+  };

 /*
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
+unsigned int defaultfg = 15;
 unsigned int defaultbg = 0;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
@@ -128,7 +128,7 @@
  * 6: Bar ("|")
  * 7: Snowman ("☃")
  */
-static unsigned int cursorshape = 2;
+static unsigned int cursorshape = 6;

 /*
  * Default columns and rows numbers
